torch/lib/c10d/logger.hpp

```diff
@@ -60,6 +62,8 @@ class Logger {
   std::shared_ptr<c10d::Reducer> reducer_;
   // track the number of iterations when runtime stats are collected so far.
   long num_iterations_stats_recorded_ = 0;
+  // Track whether DDP comm. hook has been set or not
+  bool comm_hook_set_{false};
```

Comment:
This field does not seem to be necessary, as its value can be determined by `ddp_logging_data_->comm_hook`.